rohrpost/message.py

```diff
@@ -14,10 +14,10 @@ def default(self, obj):
         return json.JSONDecoder.default(self, obj)
 
 
-def _send_message(*, message, content: dict, close: bool):
-    message.reply_channel.send(
-        {"text": json.dumps(content, cls=TolerantJSONEncoder), "close": close}
-    )
+def _send_message(*, consumer, content: dict, close: bool):
+    consumer.send(json.dumps(content, cls=TolerantJSONEncoder))
+    if close is True:
+        consumer.close()
```

Comment:
I don't think that this makes much sense here any more.  The caller of the utility functions (ie. a rohrpost handle) needs a reference to the consumer.  It can therefore close the connection itself.  I'd say it's too rarely used.

As an alternative to the rohrpost handler closing the connection itself, we could also close the connection depending on the return value of the handler.  That would allow multiple handlers to run before the connection is closed.  (Atm. only one handler can be defined per rohrpost `type`.

Reply:
Yes, it's rarely used and the consumer can also close the connection. We should just make a note in the migration guide that consumers are now expected to take care of connection closing explicitly.

Reply:
I'll change that outside of this PR.